crime-commons-classes/src/main/java/uk/gov/justice/laa/crime/filter/mdc/AddLaaTransactionIdToMDC.java

```diff
@@ -31,12 +24,13 @@ public void doFilter(ServletRequest request,
 
     String laaTransactionIdHeaderValue = extractLaaTransactionIdFromHeader(request);
 
-    if(StringUtils.isNotBlank(laaTransactionIdHeaderValue)){
+    if (StringUtils.isNotBlank(laaTransactionIdHeaderValue)) {
       log.debug("Adding the {} [{}] to the MDC", LAA_TRANSACTION_ID, laaTransactionIdHeaderValue);
```

Comment:
Is the logging level correct, should this be at INFO?

Reply:
I'm not sure what value having this at info level will bring, as the LAA Transaction Id will be present in the MDC (so all logs) anyway.
I included this only as a means to make it easier for future devs to debug MDC issues when running locally